ui/src/components/FormModifications/FormModifications.test.tsx

```diff
@@ -0,0 +1,238 @@
+        firstModificationField,
+        secondValueToInput,
+        firstStandardTextField.field
+    );
+    const mods2Field2 = findMods(
+        firstModificationField,
+        secondValueToInput,
+        secondStandardTextField.field
+    );
+
+    if (componentMakingModsTextBox1) {
+        await userEvent.type(componentMakingModsTextBox1, firstValueToInput);
+    }
+
+    const verifyAllProps = (
+        parentElement: Element | null | undefined,
+        input: Element | null | undefined,
+        mods?: { value?: string | number | boolean; help?: string; label?: string } | null
+    ) => {
+        expect(input?.getAttribute('value')).toEqual(mods?.value);
+        expect(parentElement?.textContent?.includes(mods?.help || 'fail')).toEqual(true);
+        expect(parentElement?.textContent?.includes(mods?.label || 'fail')).toEqual(true);
+    };
+
+    expect(componentInput).toHaveAttribute('disabled');
+    verifyAllProps(componentParentElement, componentInput, mods1Field1);
+
+    expect(component2Input).toHaveAttribute('disabled');
+    verifyAllProps(component2ParentElement, component2Input, mods1Field2);
+
+    if (componentMakingModsTextBox1) {
+        await userEvent.type(componentMakingModsTextBox1, secondValueToInput);
+    }
+
+    expect(componentInput).not.toHaveAttribute('disabled');
+    verifyAllProps(componentParentElement, componentInput, mods2Field1);
+
+    expect(component2Input).not.toHaveAttribute('disabled');
+    verifyAllProps(component2ParentElement, component2Input, mods2Field2);
+});
+
+it('verify markdown modifications', async () => {
+    setUpConfigWithDefaultValue();
+    renderModalWithProps(props);
+
+    const firstValueToInput = 'a';
+    const secondValueToInput = 'aa';
+
+    const [componentParentElement] = getHTMLTextComponentsForField(firstStandardTextField.field);
+    const [component2ParentElement] = getHTMLTextComponentsForField(secondStandardTextField.field);
+
+    const componentMakingModsTextBox1 = document.querySelector(
+        `.${secondModificationField.field} input`
+    );
+
+    const firstElementOuterHTMLBeforeMods = componentParentElement?.outerHTML;
+
+    const mods1Field2 = findMods(
+        secondModificationField,
+        firstValueToInput,
+        secondStandardTextField.field
+    );
+
+    const mods2Field1 = findMods(
+        secondModificationField,
+        secondValueToInput,
+        firstStandardTextField.field
+    );
+    const mods2Field2 = findMods(
+        secondModificationField,
+        secondValueToInput,
+        secondStandardTextField.field
+    );
+
+    if (componentMakingModsTextBox1) {
+        await userEvent.type(componentMakingModsTextBox1, firstValueToInput);
+    }
+
+    const firstElementOuterHTMLAfterMods = componentParentElement?.outerHTML;
+
+    // only id specified as modification so no changes should occure
+    expect(firstElementOuterHTMLBeforeMods).toEqual(firstElementOuterHTMLAfterMods);
+
+    expect(
```

Comment:
that's too complicated. Maybe to replace it with simple but fragile
```ts
expect(component2ParentElement.textContent).toMatchInlineSnapshot();
```

Reply:
changed